ocfweb/static/scss/pages/docs/_servers.scss

```diff
@@ -69,6 +69,13 @@
                     }
                 }
             }
+
+            .server-anchor {
+                display: block;
+                position: relative;
+                top: -75px;
```

Comment:
Where does this `-75px` come from? Does this work with the responsive design of the page? I'm assuming it does, but I'm just not sure where this number comes from.